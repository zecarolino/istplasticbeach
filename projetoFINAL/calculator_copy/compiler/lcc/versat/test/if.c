int x = 4, y = 4, z = 4;
int main() {
	int s = 0;
	if (x == y) {
		if (y == z) s = 3;
		else s = 2;
	}
	else
	if (z == y) {
		s = 2;
	}
	else
	if (z == x) {
		s = 2;
	}
	else s = 1;
	return s;
}
