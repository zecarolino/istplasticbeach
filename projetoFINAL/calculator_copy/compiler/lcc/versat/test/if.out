VCD info: dumpfile xtop.vcd opened for output.
3
Catched Trap at time  457 clock cycles:
